languages/python/sqlalchemy-oso/sqlalchemy_oso/session.py

```diff
@@ -11,6 +11,29 @@
 from sqlalchemy_oso.auth import authorize_model_filter
 
 
+class OsoSession:
```

Comment:
Make this a private class. Either:

- define in `set_get_session` 
- name `_OsoSession`.